backstage/vendor_images.yaml

```diff
@@ -0,0 +1,4 @@
+ghcr.io:
+  images:
+    backstage/backstage:
```

Comment:
could we use the regex format to start vendoring future vsns too?

Reply:
Yes, fixed